src/View/Form/EntityContext.php

```diff
@@ -649,7 +648,6 @@ protected function _getTable($parts, $fallback = true): ?Table
                     continue;
                 }
             } else {
-                /** @var \Cake\ORM\AssociationCollection $associationCollection */
```

Comment:
Same maybe here.

Reply:
The `$table` var is already annotated as `ORM\Table` earlier in the method and return type of `associations()` method as `AssociationCollection` typehint, so this annotation is redundant.